fuzzers/forkserver/forkserver_libafl_cc/src/lib.rs

```diff
@@ -1,15 +1,20 @@
+use libafl_bolts::shmem::{ShMemProvider, StdShMemProvider};
 use libafl_targets::{
     map_input_shared_memory, map_shared_memory, start_forkserver, MaybePersistentForkserverParent,
 };
 
 #[no_mangle]
 pub extern "C" fn libafl_start_forkserver() {
+    let Ok(mut shm_provider) = StdShMemProvider::new() else {
```

Comment:
On Android StdShMemProvider spawns a server, we need to double check if this works for this use case (otherwise we should set a different provider here)

Reply:
I totally agree. But I know little about Android, and I also don't know why the standard shmem provider needs a service wrapper. Maybe this should be checked by someone knowing this detail.

Reply:
We need a service on android to hand out ashmem handles via unix sockets, as that is the only way to get a 'reference' to an existing shared memory allocation.

Reply:
This is for multi process IPC. For a forkserver we don't need that since the child can just inherit the allocations on fork

Reply:
True.

Reply:
@Evian-Zhang probably we want to hard-code AshMemProvider here

Reply:
Hmmm, is it possible for someone to setup a android fuzzer test CI so that we can test the correctness? I personally don't know any of android things, so although I'm willing to hard code it, I'm afraid we cannot ensure its correctness.

Reply:
I will do that shortly these days. Will ping you if there is some failures.

Considering the slow performance of QEMU emulation, maybe we can just test simple fuzzers.

Reply:
Thank you! I will improve this shmem after your work :)